### PR TITLE
add recipe for ldsc

### PR DESCRIPTION
**Pull request by hmenager**
ldsc is a command line tool for estimating heritability and genetic
correlation from GWAS summary statistics. ldsc also computes LD Scores.

Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by hmenager** [comment]
Should be fine by now @abretaud, thanks a lot for your help!

File: recipes/ldsc/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -eu -o pipefail
+wget https://patch-diff.githubusercontent.com/raw/bulik/ldsc/pull/176.patch
```
Comment (abretaud) [inline recipes/ldsc/build.sh]: Could you add the 176.patch to the recipe folder instead? In case it's not online anymore some day

Comment (hmenager) [reply recipes/ldsc/build.sh]: I'm on it, thanks

Comment (dpryan79) [reply recipes/ldsc/build.sh]: You can get rid of `wget` in the recipe then :)

**Comment by dpryan79** [comment]
I've asked upstream to tag a new version, which would make this process rather simpler.

**Comment by dpryan79** [comment]
I'm not sure how responsive upstream will be to tagging a new release, so if you need this soonish then just let me know and we can merge this in and deal with a new 1.0.1 release when/if it comes.

**Comment by hmenager** [comment]
Thank you for your help @dpryan79. If it isn't too much trouble, It would be super nice to merge this soon, because we need it for submission. Additionally, I'm really not sure about upstream's responsiveness, I haven't yet received any feedback on my PR, and the repository owner seems to have academia.

**Comment by dpryan79** [comment]
Can't argue with that, let's merge now then :)

**Pull request merged**